cmd/query/app/http_handler_test.go

```diff
@@ -329,9 +329,6 @@ func TestGetTrace(t *testing.T) {
 			require.NoError(t, err)
 			assert.Empty(t, response.Errors)
 
-			assert.Len(t, exporter.GetSpans(), 1, "HTTP request was traced and span reported")
-			assert.Equal(t, "/api/traces/{traceID}", exporter.GetSpans()[0].Name)
```

Comment:
the setup of this test doesn't make sense then - it's explicitly creates internal tracer/exporter/etc. So is it testing the behavior of the handler, or the fact that the handler itself is being traced? If it does both then we probably need to move the "traced" part into `server_test.go`

Reply:
@yurishkuro moved this test to the handler. It looks like the span name handler was required.

Reply:
also - with the current setup, the old flow won't be traced anymore. is that fine? do we want a test for that as well?

Reply:
Can we move this test to some other place, e.g. at the top Server level? We want to make sure the handlers are still traced, and that the span name is not high cardinality.

Reply:
@yurishkuro We already this in https://github.com/jaegertracing/jaeger/pull/6131. Are you asking about something different?